components/Header.tsx

```
@@ -18,11 +23,11 @@ export default function Header() {
             headingLevel="h1"
             classname="font-space-grotesk font-bold text-5xl text-accent-red"
           >
-            Saurav Khare
+            {data && data[0].heading}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)